src/python/grpcio_tests/setup.py

```diff
@@ -80,8 +80,8 @@
         'credentials/server1.key',
         'credentials/server1.pem',
     ],
-    'tests.protoc_plugin': [
-        'protoc_plugin_test.proto',
```

Comment:
I don't believe the file exists in our code-base anymore. I think it was split it up at some point into the multiple proto files under `grpcio_tests/src/tests/protoc_plugin/protos`. Furthermore, those files that it was split up into are handled by the setuptools custom command as opposed to needing to be available on installation, so they never needed to be added back to the package data. OTOH, the `split_test.proto` *does* because we need to use the old-style test of invoking `protoc` on it at test time in order to use the split-up-the-generated-source-between-multiple-directories-and-generate-arbitrary-trash approach.